node/cn/api_backend.go

```diff
@@ -278,8 +278,14 @@ func (b *CNAPIBackend) ProtocolVersion() int {
 	return b.cn.ProtocolVersion()
 }
 
+// If current block is magma hard forked, it returns the baseFee * 2.
+// Other cases, it returns the unitPrice.
 func (b *CNAPIBackend) SuggestPrice(ctx context.Context) (*big.Int, error) {
-	return b.gpo.SuggestPrice(ctx)
+	suggestedPrice, _ := b.gpo.SuggestPrice(ctx)
```

Comment:
May I ask why you added this logic here rather than `b.gpo.SuggestPrice(ctx)` ?

Reply:
If so, maybe we can add one more parameter like `isMagmaForkEnabled` so pass whether after Magma or not?

Reply:
That's possible. Actually gpo has backend instance by interface so If we add the currentBlock() in the interface then we can get the block number.
How do you think about this?

Reply:
Sounds good, or we can use txPool because tx pool also has fork indicator.

Reply:
Good idea! I think It's better to use the txPool's field.

Reply:
Ohh... but It needs an additional function like isMagma().
I think hard fork checking function, isMagmaCompatible(), is enough to avoid confusion so we can just use the currentBlock().

Reply:
Okay~ whatever you think is better, i'm down for it!